packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -1,16 +1,137 @@
 
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development. They are subject to non-backward compatible changes or removal in any future version. These are not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be announced in the release notes. This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
 ---
 <!--END STABILITY BANNER-->
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+Amazon CloudWatch Synthetics allow you to monitor your application by generating **synthetic** traffic. The traffic is produced by a **canary**: a configurable script that runs on a schedule. You configure the canary script to follow the same routes and perform the same actions as a user, which allows you to continually verify your user experience even when you don't have any traffic on your applications.
+
+## Canary
+
+To illustrate how to use a canary, assume your application defines the following endpoint:
+
+```bash
+% curl "https://api.example.com/user/books/topbook/"
+The Hitchhikers Guide to the Galaxy
+
+```
+
+The below code defines a canary that will hit the `books/topbook` endpoint every 5 minutes: 
 
 ```ts
 import * as synthetics from '@aws-cdk/aws-synthetics';
+
+const canary = new synthetics.Canary(this, 'MyCanary', {
+  schedule: synthetics.Schedule.rate(Duration.minutes(5)),
+  test: Test.custom({
+    code: Code.fromAsset(path.join(__dirname, 'canary'))),
+    handler: 'index.handler',
+  }),
+});
+```
+
+The following is an example of an `index.js` file which exports the `handler` function, note that the function **must** be called `handler`:
+
+```js
+var synthetics = require('Synthetics');
+const log = require('SyntheticsLogger');
+
+const pageLoadBlueprint = async function () {
+
+    // INSERT URL here
+    const URL = "https://api.example.com/user/books/topbook/";
+
+    let page = await synthetics.getPage();
+    const response = await page.goto(URL, {waitUntil: 'domcontentloaded', timeout: 30000});
+    //Wait for page to render.
+    //Increase or decrease wait time based on endpoint being monitored.
+    await page.waitFor(15000);
+    await synthetics.takeScreenshot('loaded', 'loaded');
+    let pageTitle = await page.title();
+    log.info('Page title: ' + pageTitle);
+    if (response.status() !== 200) {
+        throw "Failed to load page!";
+    }
+};
+
+exports.handler = async () => {
+    return await pageLoadBlueprint();
+};
+```
+
+The canary will automatically produce a CloudWatch Dashboard:
+
+![UI Screenshot](images/ui-screenshot.png)
+
```

Comment:
Following up on the comments above, we can add a line here that elborate on the Synthetics capabilities:

```
The Canary code will be executed in a lambda function created by Synthetics on your behalf. The Lambda function includes a custom [runtime](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_Library.html) provided by Synthetics. The provided runtime includes a variety of handy tools such as [Puppeteer](https://www.npmjs.com/package/puppeteer-core)  and Chromium. To learn more about Synthetics capabilities, check out the [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html)
```